server/src/main/scala/org/http4s/server/middleware/CSRF.scala

```diff
@@ -85,7 +84,7 @@ final class CSRF[F[_], G[_]] private[middleware] (
       lift(joined + "-" + encodeHexString(out))
     }
 
-  /** Generate a new token **/
+  /** Generate a new token * */
```

Comment:
Looks like this gap (and a few similar below) between two 'star' symbols is due to improved scalafmt rules. Would it  make sense to remove these redundant stars?

Reply:
Yes, we could and probably should clean those up.  Stand back, I have emacs.